src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestResponseCode.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+public enum StreamingIngestResponseCode {
```

Comment:
I dont see this class being used. Am I missing something?

Also, few of the error codes are kind of cryptic to customers, are we making them all visible?

Reply:
@sfc-gh-japatel they're used in `getMessage`

And +1, lots of the errors don't make sense to be exposed, for example ERR_ENQUEUE_TABLE_CHUNK_QUEUE_FULL or ERR_INVALID_COLUMN_IN_EP_INFO